sdk/monitor/monitor-opentelemetry/README.md

```diff
@@ -82,23 +82,22 @@ const options: AzureMonitorOpenTelemetryOptions = {
   logRecordProcessors: [],
   spanProcessors: [],
 };
-
 useAzureMonitor(options);
 ```
 
-|Property|Description|Default|
```

Comment:
Whatever formatting this is doing breaks the table here.

Reply:
Redid it in pure HTML to get rid of the issues with formatting.